src/event.rs

```diff
@@ -663,6 +663,21 @@ pub struct Touch {
     pub force: Option<Force>,
     /// Unique identifier of a finger.
     pub id: u64,
+    /// Type of pointer used when touching.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - Currently only available on Web targets.
+    pub pointer_type: Option<PointerType>,
```

Comment:
```suggestion
```

Is it truly necessary to expose this now?
Adding a richer pointer API has been discussed in #99 and #336, but I don't think this is the right time (or place) to add it.

Reply:
Fair enough, I've remove it and any reference to it for now.